README.en.md

```diff
@@ -308,6 +308,8 @@ NOT SUPPORTED
 }
 ```
 
+*New in 1.8.4*: `url` can contain a port number, e.g. `www.site.com:8081/english`.
```

Comment:
I don't think we need to start adding changelogs to the readme?

Reply:
Hmm, but it would be very confusing for people if we don't specify a version, wouldn't it?

Reply:
I mean each time we have added new features we have been adding to this readme, which won't apply to people that are using previous versions. I just noticed that we mentioned version number before in 
>Starting from version 1.3.0, you must create the configuration file in JSON format if you want to make use of the `custom_domain` URL pattern.
which _maybe_ feels OK since it is a major change, but I think if we start doing this for every minor change (new in 1.8.6, 1.8.7...), it's going to be even more confusing.

Reply:
Yeah that's a valid point. But I think we still need to make changes visible and clear to end users. Maybe put this in swapperdocs?

Reply:
That's what the github releases page is for